packages/interim/src/renovation/component_wrapper/common/mutations_recording.ts

```diff
@@ -0,0 +1,27 @@
+function isChildNode(node: Node): node is ChildNode {
+    return typeof (node as Partial<ChildNode>).remove === 'function';
+  }
+  
+  function revertMutation({ type, addedNodes }: MutationRecord): void {
+    switch (type) {
```

Comment:
Minor: Possible if is more simple here instead of stwitch

Reply:
I prefer using explicit `switch` in such cases because:
1) less changes in the future if you need to process the other cases (no need to change `if` to `switch`)
2) it is emphasised that there can be more then two branches